### DISCUSSION TITLE
phpstorm hover suggestions autocompletions

### DISCUSSION THREAD

**Discussion by CamKem** [tailwindlabs/tailwindcss#10540] [Help]
Hi all, I usually use laravel + vue and tailwind, which when I start a new project tailwind autocompletes show up & work perfectly.
I am doing a project where I am working with vanilla html & have linked tailwind via a CDN for dev, but the autocompletes are not coming up. How do I set it up so phpstorm will give me the autocompletion suggestions on over?

**Comment by wongjn** [comment]
Ensure that there is a `tailwind.config.js` in the root of the project.
